packages/gnucobol/build.sh

```diff
@@ -2,20 +2,36 @@ TERMUX_PKG_HOMEPAGE=https://gnucobol.sourceforge.io/
 TERMUX_PKG_DESCRIPTION="A free/libre COBOL compiler"
 TERMUX_PKG_LICENSE="GPL-3.0, LGPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.1.2
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=3.2
+TERMUX_PKG_REVISION=4
```

Comment:
If you increase the version then revision is to be set to 0 or 1 (please check the docs)